packages/-ember-data/node-tests/fixtures/expected.js

```diff
@@ -252,5 +252,15 @@ module.exports = {
     'pushPayload [OPTIONAL]',
     'serializeIntoHash [OPTIONAL]',
     'normalize [OPTIONAL]',
+    'columnNameToDesc',
```

Comment:
What caused this change?

Reply:
fixing the inclusion of the `debug` package, there is a follow up PR to this on that will make our tests and fixtures show the module, class, package and public/private status of each piece of API documentation so that "why" something changes is more clear.

Reply:
Can we add that to the PR description please?

Reply:
added

Reply:
Thanks!